### ISSUE TITLE
💡 [FEATURE] - <Adding a section for Data Analytics>

### ISSUE THREAD

**Issue by MIpaltan007** [jfmartinz/ResourceHub#334]
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

Data Analytics is one of the most demanding skill in the present job market, after the likes of AI/ML. So its important to have a section dedicated to Data Analytics, helping all the aspiring Data Analysts

### Explain your solution

Being an aspiring Data Analyst myself, i can provide useful study materials, course links(YouTube channels) and resources for learning this domain and its application.

**Comment by MIpaltan007** [comment]
You can assign me this @jfmartinz . I won't disappoint you this time

**Comment by jfmartinz** [comment]
Is this new category like web development? No worries :)

**Comment by MIpaltan007** [comment]
Yes @jfmartinz. It is a domain which involves Computer Science with Mathematics(Statistics to be precise). Its a subset of Data Science.

**Comment by jfmartinz** [comment]
Can you instead help me create some sections or resources. Like for example:
- [App Development](https://github.com/jfmartinz/ResourceHub/tree/main/App%20Development) lacks some sections. You can take a look to other domains like web development, etc to get an idea what to add other section to App Development.
- [Data Science](https://github.com/jfmartinz/ResourceHub/tree/main/Data%20Science)
- etc etcc

I can't add another domain since, I think the domains added are enough for now. My plain is to add moreresources or section to these domain first. Could you help me? YOu can create issues and I will asssign it to yo

**Comment by jfmartinz** [comment]
Is Data Analytics and Data Science not the same?

**Comment by MIpaltan007** [comment]
Data Analytics is within the domain of Data Science. Ok, I'll try to add something to App Dev, Data Science.

**Comment by jfmartinz** [comment]
Sure. Thanks. If this is the same then [Data Science](https://github.com/jfmartinz/ResourceHub/tree/main/Data%20Science) exist already, but you can add sections in this domain.

**Comment by MIpaltan007** [comment]
@jfmartinz  I have added the resources, Please merge this PR. Also add the gssoc label along with level label.
